lib/active_shipping/carriers/canada_post_pws.rb

```diff
@@ -294,6 +294,16 @@ def parse_rates_response(response, origin, destination)
       CPPWSRateResponse.new(true, "", {}, :rates => rates)
     end
 
+    def price_from_node(node, exclude_tax)
+      price = node.at('price-details/due').text
+      return price unless exclude_tax
+      taxes = 0
+      node.at('price-details/taxes').children.each do |node|
+        taxes += Package.cents_from(node.text) if node.elem?
```

Comment:
nit: there's also a cleaner way to update `taxes` without doing a side effect in a block.

```ruby
children = node.at('price-details/taxes').children
elem_nodes = children.select { |node| node.elem? }
taxes = elem_nodes.sum { |node| Package.cents_from(node.text) }
```